tim_sort/Python/tim_sort.py

```diff
@@ -0,0 +1,13 @@
+
+def tim_sort(arr):  # Function that performs the tim sort
+   arr.sort()
+
+
+def main():
+    arr = [12, 96, 24, 48, 36, 60, 84, 72]    # Declaring a sample array
+    tim_sort(arr)  # Calling the bubbleSort function and passing the array
+    print('Sorted array is: {}'.format(' '.join(map(str, arr))))
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
+++ b/tmp/tmpc083vmnv/tim_sort/Python/tim_sort.py
@@ -10,4 +10,4 @@
 
 
 if __name__ == '__main__':
-    main()+    main()
```